mne/decoding/time_gen.py

```diff
@@ -30,9 +30,10 @@ class _DecodingTime(dict):
         seconds). Defaults to one time sample.
     'length' : float
         Duration of each classifier (in seconds). Defaults to one time sample.
-    If None, empty dict. Defaults to None."""
+    If None, empty dict. """
 
     def __repr__(self):
+        """Human readable representation"""
```

Comment:
I know @mainakjas made the comment to add this, but is this at all necessary. none of our other built-in functions have a description and none of the other __repr__ overrides.